multiqc/modules/odgi/odgi.py

```diff
@@ -0,0 +1,277 @@
+        )
+
+        self.odgi_stats_map = dict()
+
+        # Find and load any odgi stats file
+        log_files = self.find_log_files("odgi", filehandles=True)
+
+        # Parse odgi stats data
+        for f in log_files:
+            self.parse_odgi_stats_report(f)
+            self.add_data_source(f)
+
+        self.odgi_stats_map = self.ignore_samples(self.odgi_stats_map)
+
+        if not self.odgi_stats_map:
+            raise UserWarning
+
+        if len(self.odgi_stats_map) == 1:
+            log.info("Found {} report".format(len(self.odgi_stats_map)))
+        else:
+            log.info("Found {} reports".format(len(self.odgi_stats_map)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.odgi_stats_map, "multiqc_odgi_stats")
+
+        # Add the general section containing the general odgi stats
+        self.plot_general_odgi_stats()
+
+        # Plot the odgi stats metrics as a lineplot
+        self.plot_odgi_metrics()
+
+    def parse_odgi_stats_report(self, f):
+        """
+        Load the odgi stats YAML file.
+        The stats reports assume the following YAML structure:
+
+        ---
+        length: 206263
+        nodes: 3751
+        edges: 5195
+        paths: 13
+        num_weakly_connected_components: 1
+        weakly_connected_components:
+          - component:
+              id: 0
+              nodes: 3751
+              is_acyclic: 'no'
+        num_nodes_self_loops:
+          total: 0
+          unique: 0
+        A: 57554
+        C: 43275
+        G: 41944
+        T: 63490
+        mean_links_length:
+          - length:
+              path: all_paths
+              in_node_space: 1.64973
+              in_nucleotide_space: 7.34035
+              num_links_considered: 202793
+              num_gap_links_not_penalized: 147940
+        sum_of_path_node_distances:
+          - distance:
+              path: all_paths
+              in_node_space: 5.53383
+              in_nucleotide_space: 2.1454
+              nodes: 202806
+              nucleotides: 3757597
+              num_penalties: 231
+              num_penalties_different_orientation: 0
+        """
+        data = {}
+        try:
+            data = yaml.load(f["f"], Loader=yaml.SafeLoader)
+        except Exception as e:
+            log.warning("Could not parse YAML for '{}': \n  {}".format(f, e))
+        self.odgi_stats_map.update(self.compress_stats_data(self.extract_sample_name(f["fn"]), data))
+
+    def plot_general_odgi_stats(self):
+        """
+        Plot the odgi stats by adding them to the general statistics table.
+        """
+        data = dict()
+        headers = OrderedDict()
+        headers["Length"] = {"title": "Length", "description": "Graph length in nucleotides.", "scale": "BuPu"}
+        headers["Nodes"] = {
+            "title": "Nodes",
+            "description": "Number of nodes in the graph.",
+            "scale": "OrRd",
```

Comment:
As the number of nodes will always be an integer I assume, you could do a format here and remove the `.0` - same for the other columns. 

`'format': '{:,.0f}',` should do the trick here 👍🏻

Reply:
I was already wondering how to fix this, thanks for your suggestions!

Reply:
Worth mentioning that generally I try to make big numbers small. It's tough to compare `18392833` against `1839283` but it's easy to compare `18.3` against `1.8`. So if you expect a column to have large numbers in it's usually better to keep the decimals and use the `modify` key to reduce the order.

Docs: https://multiqc.info/docs/#step-3-adding-to-the-general-statistics-table

https://github.com/ewels/MultiQC/blob/21875074c6cbcf8084ea7017522f7cde04b269b5/docs/modules.md#L675-L684

Reply:
I understand, but the numbers for different kind of versions of the graphs might be different by a huge factor. Or a tiny factor. And for validation and debugging, I wanted to see both.